autoload/LanguageClient.vim

```diff
@@ -201,6 +209,15 @@ function! s:inputlist(...) abort
     return l:selection
 endfunction
 
+function! s:selectionUI_funcref(source, sink) abort
+    if get(g:, 'LanguageClient_selectionUI', 'FZF') ==? 'FZF'
+                \ && get(g:, 'loaded_fzf')
+        call s:FZF(a:source, a:sink)
+    else
```

Comment:
There's still an issue here, sorry I didn't spot it before. If you set `let g:LanguageClient_selectionUI = 'location-list'` it will error, because it will jump to the else branch and that expects a `Funcref`.

Y

Reply:
You'll probably need something similar to what's done in `contextMenu` which opens `inputlist` if it can't use FZF.

Reply:
Wow, this is turning out to be much more confusing than I had anticipated. Sorry *again* for messing up.

I guess I'll have to make a table of the possible interactions.

Reply:
Wait—that case shouldn't happen at all. `s:selectionUI_funcref()` is only called from the `SelectionUI::Funcref` codepaths in Rust, which would not be hit if `g:LanguageClient_selectionUI = 'location-list'` or `'quickfix'`. Right?

Reply:
Ah wait, it's an issue with code actions and code lens actions only. This is rather unfortunate, but maybe I can try fixing that before merging this one, let me see if I can push that through soon enough for you to rebase.